packages/hint/src/lib/hint-context.ts

```diff
@@ -19,6 +19,8 @@ import { ProblemLocation, Severity } from '@hint/utils/dist/src/types/problems';
 import { Category } from '@hint/utils/dist/src/types/category';
 import { getHTMLCodeSnippet } from '@hint/utils/dist/src/report/get-html-code-snippet';
 
+export type CodeLanguage = 'css' | 'html' | 'http' | 'javascript';
```

Comment:
Are these the strings we can pass directly to our syntax highlighter? Or will we need to perform some mapping?

Reply:
they should be the strings to pass directly to the syntax highlighter